libraries/Arduino_LED_Matrix/examples/TextWithArduinoGraphics/TextWithArduinoGraphics.ino

```diff
@@ -1,3 +1,6 @@
+// TextAnimation works only when ArduinoGraphics is installed and used.
+// ArduinoGraphics is an external library and needs to be installed manually
+// using library manager.
```

Comment:
```suggestion
// ArduinoGraphics is an external library and needs to be installed using
// Library Manager.
```

The term "manually" when used in the context of library installation has a meaning that is contradictory to what this comment is intended to communicate:

https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#manual-installation